src/ROCrate/Generic/File.fs

```diff
@@ -0,0 +1,123 @@
+
+    static member name = "http://schema.org/name"
+
+    static member comment = "http://schema.org/comment"
+
+    static member disambiguatingDescription = "http://schema.org/disambiguatingDescription"
+
+    static member usageInfo = "http://schema.org/usageInfo"
+
+    static member encodingFormat = "http://schema.org/encodingFormat"
+
+    static member tryGetNameAsString(dt : LDNode, ?context : LDContext) =
+        match dt.TryGetPropertyAsSingleton(File.name, ?context = context) with
+        | Some (:? string as n) -> Some n
+        | _ -> None
+
+    static member getNameAsString(dt : LDNode, ?context : LDContext) =
+        match dt.TryGetPropertyAsSingleton(File.name, ?context = context) with
+        | Some (:? string as n) -> n
+        | Some _ -> failwith $"Property of `name` of object with @id `{dt.Id}` was not a string"
+        | _ -> failwith $"Could not access property `name` of object with @id `{dt.Id}`"
+
+    static member setNameAsString(dt : LDNode, name : string, ?context : LDContext) =
+        dt.SetProperty(File.name, name, ?context = context)
+
+    static member getComments(dt : LDNode, ?graph : LDGraph, ?context : LDContext) =
+        let filter ldObject context = Comment.validate(ldObject, ?context = context)
+        dt.GetPropertyNodes(File.comment, filter = filter, ?graph = graph, ?context = context)
+
+    static member setComments(dt : LDNode, comment : ResizeArray<LDNode>, ?context : LDContext) =
+        dt.SetProperty(File.comment, comment, ?context = context)
+
+    static member tryGetDisambiguatingDescriptionAsString(dt : LDNode, ?context : LDContext) =
+        match dt.TryGetPropertyAsSingleton(File.disambiguatingDescription, ?context = context) with
+        | Some (:? string as dd) -> Some dd
+        | _ -> None
+
+    static member setDisambiguatingDescriptionAsString(dt : LDNode, disambiguatingDescription : string, ?context : LDContext) =
+        dt.SetProperty(File.disambiguatingDescription, disambiguatingDescription, ?context = context)
+
+    static member tryGetEncodingFormatAsString(dt : LDNode, ?context : LDContext) =
+        match dt.TryGetPropertyAsSingleton(File.encodingFormat, ?context = context) with
+        | Some (:? string as ef) -> Some ef
+        | _ -> None
+
+    static member setEncodingFormatAsString(dt : LDNode, encodingFormat : string, ?context : LDContext) =
+        dt.SetProperty(File.encodingFormat, encodingFormat, ?context = context)
+
+    static member tryGetUsageInfoAsString(dt : LDNode, ?context : LDContext) =
+        match dt.TryGetPropertyAsSingleton(File.usageInfo, ?context = context) with
+        | Some (:? string as ui) -> Some ui
+        | _ -> None
+
+    static member setUsageInfoAsString(dt : LDNode, usageInfo : string, ?context : LDContext) =
+        dt.SetProperty(File.usageInfo, usageInfo, ?context = context)
+
+    static member genId(name : string) =
+        $"{name}"
+
+    static member validate(dt : LDNode, ?context : LDContext) =
```

Comment:
Is this the equivalent of the missing file profile for https://github.com/nfdi4plants/isa-ro-crate-profile/issues/57?

Reply:
Yes

Reply:
... I mean yesn't. 

It isn't missing, File = [Data](https://github.com/nfdi4plants/isa-ro-crate-profile/blob/release/profile/isa_ro_crate.md#data)